src/containers/pages/IRS/assignments/index.tsx

```diff
@@ -167,7 +150,11 @@ interface DispatchedStateProps {
 /** map state to props */
 const mapStateToProps = (state: Partial<Store>): DispatchedStateProps => {
   const planStatus = [PlanStatus.ACTIVE];
-  const plans = getPlanRecordsArray(state, InterventionType.IRS, planStatus);
+  const fiPlans = getPlanRecordsArray(state, InterventionType.FI, planStatus);
+  const irsPlans = getPlanRecordsArray(state, InterventionType.IRS, planStatus);
```

Comment:
Currently we have to fetch these independently by plan type, should have ticket to add functionality to get all plans regardless of type (currently a type is required).

Reply:
Issue created at https://github.com/onaio/reveal-frontend/issues/547